parts/linux/cloud-init/artifacts/components.json

```diff
@@ -833,7 +833,7 @@
               {
                 "k8sVersion": "1.29",
                 "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
-                "latestVersion": "v1.29.2"
+          "latestVersion": "v1.29.11"
```

Comment:
missing a tab here.

Reply:
oh right. Will update the template to include additional tab.
Is it good to cache this version?

Reply:
closing this PR for Renovate to create another one with correct indent.